entries/p/paypal.com.json

```diff
@@ -10,7 +10,7 @@
       "totp",
       "u2f"
     ],
-    "notes": "Some methods of 2FA are only available in a few countries. Contact PayPal for more information.",
+    "notes": "Some methods of 2FA are only available in a few countries. Contact PayPal for more information. (2fa isn't usefull for paypal as they default to single factor sms)",
```

Comment:
Looks like TOTP is also supported as a backup method.
![](https://user-images.githubusercontent.com/3535780/198904639-d42eba84-8806-41fc-a87a-d9ae2f70317c.png)
```suggestion
    "notes": "Some methods of 2FA are only available in a few countries. Contact PayPal for more information. U2F/WebAuthn requires a backup authentication method",
```

Reply:
@Carlgo11 I realized  I need to give more context paypal current state doesn't actually meet the requirements of 2fa as they allow you to login via sms as the only factor on any device even if it wasn't previously used. This was my experience yesterday that lead me to closing my account

Reply:
@person1234-ux I tried to reproduce what you described but the result differs from yours.

If I enter my correct password I get the following options:
![Screenshot 2022-10-31 at 00-00-09 PayPal](https://user-images.githubusercontent.com/3535780/198906321-38614337-ffb1-4e3f-a322-678a02753a84.png)

If I click on _forgot password_ I indeed get the option of SMS authentication.
If we're to assume that factor gets compromised (due to SIM-swapping) I still get the following requirement after successfully entering the SMS code:
![Screenshot 2022-10-31 at 00-07-17 PayPal](https://user-images.githubusercontent.com/3535780/198906420-aeede1d3-797a-4156-a6e6-cfdcb8f22b32.png)

So by that process the authentication factors are the same but the methods are weaker:
|Circumstance|Knowledge method|Possession method|
|-|-|-|
|Normal Authentication|Password|U2F Key|
|Password Reset|Security Questions|SMS code|

Of course SMS + Questions are far from ideal but technically 2FA is still enforced.